tools/src/main/java/org/apache/kafka/tools/GetOffsetShell.java

```diff
@@ -107,12 +107,7 @@ private static class GetOffsetShellOptions extends CommandDefaultOptions {
         public GetOffsetShellOptions(String[] args) throws TerseException {
             super(args);
 
-            OptionSpec<String> brokerListOpt = parser.accepts("broker-list", "DEPRECATED, use --bootstrap-server instead; ignored if --bootstrap-server is specified. The server(s) to connect to in the form HOST1:PORT1,HOST2:PORT2.")
-                    .withRequiredArg()
-                    .describedAs("HOST1:PORT1,...,HOST3:PORT3")
-                    .ofType(String.class);
             OptionSpec<String> bootstrapServerOpt = parser.accepts("bootstrap-server", "REQUIRED. The server(s) to connect to in the form HOST1:PORT1,HOST2:PORT2.")
```

Comment:
Please rename `effectiveBrokerListOpt` to `bootstrapServerOpt`. Also, please initialize it in line#110

Reply:
@chia7712 Thanks for review, updated.